systemd/afterburn-sshkeys@.service.in

```diff
@@ -8,6 +8,7 @@ ConditionKernelCommandLine=|ignition.platform.id=aliyun
 ConditionKernelCommandLine=|ignition.platform.id=aws
 ConditionKernelCommandLine=|ignition.platform.id=azure
 ConditionKernelCommandLine=|ignition.platform.id=digitalocean
+ConditionKernelCommandLine=|ignition.platform.id=exoscale
```

Comment:
I added this line to trigger the condition, and build FCOS with this file overided:
```
overrides
├── rootfs
│   └── usr
│       ├── bin
│       │   └── afterburn
│       └── lib
│           ├── dracut
│           │   └── modules.d
│           │       └── 30ignition
│           │           └── ignition
│           └── systemd
│               └── system
│                   └── afterburn-sshkeys@.service
└── rpm
``` 

and so I got this error:
```
[systemd]
Failed Units: 1
  afterburn-sshkeys@@DEFAULT_INSTANCE@.service
```

should be `afterburn-sshkeys@.service.in` instead of `afterburn-sshkeys@.service`?

Reply:
In order to run it manually, the service is called `afterburn-sshkeys@core.service`.
When it fails, journal logs should tell you what's going on.

Reply:
thanks